### ISSUE TITLE
Route pattern "/*path" contains a reserved stash value

### ISSUE THREAD

**Issue by eserte** [preaction/Statocles#604]
t/command/daemon.t and t/command/mojo_app.t fail on my smoker systems:
```
#   Failed test 'No tests run for subtest "listen on a random port"'
#   at t/command/daemon.t line 66.
Route pattern "/*path" contains a reserved stash value at /home/cpansand/.cpan/build/2021032010/Statocles-0.097-p9lIVn/blib/lib/Statocles/Command/daemon.pm line 204.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 1.
t/command/daemon.t ............ 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/1 subtests 
...
#   Failed test 'No tests run for subtest "root site"'
#   at t/command/mojo_app.t line 97.
Route pattern "/*path" contains a reserved stash value at /home/cpansand/.cpan/build/2021032010/Statocles-0.097-p9lIVn/blib/lib/Statocles/Command/daemon.pm line 204.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 1.
t/command/mojo_app.t .......... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/1 subtests 
```

Just a guess: maybe it fails with newer Mojolicious versions?

**Comment by ferki** [comment]
> Just a guess: maybe it fails with newer Mojolicious versions?

Yes, it seems to start failing with >=Mojolicious-9.0.

**Comment by preaction** [comment]
This will be fixed by the v0.098 release, shortly. Thanks for reporting this!
